recipe/meta.yaml

```diff
@@ -124,6 +124,7 @@ requirements:
     - tk                                 # [unix]
     - ncurses                            # [unix]
     - libffi                             # [unix]
+    - libnsl                             # [linux]
```

Comment:
Looks good to me but I'd be more confident if @isuruf could take a quick look before we merge.

Reply:
That's fine, but note that we don't support python 3.6 anymore.

Reply:
Yeah. I still think having a new py36 build is good for bugfixes. But building new packages for py36 is a no-no due to CI usage.